### PR TITLE
Fixes #27

### LINKED ISSUE #27: Entire page paints on scrolling
In Chrome 41, similar to Polymer/core-scroll-header-panel#38.

### PR DESCRIPTION
**Pull request by ebidel**
R: @frankiefu

### CODE REVIEW

**Comment by ebidel** [comment]
cc @ssorallen

File: core-header-panel.css [hunk 1]
```diff
@@ -31,6 +31,10 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   -webkit-overflow-scrolling: touch;
 }
 
+#mainContainer {
+  transform: translateZ(0);
```
Comment (ssorallen) [inline core-header-panel.css]: This should also have a `-webkit-transform` version before it. Safari, Mobile Safari, and Android Browser still use the prefix: http://caniuse.com/#search=transform

**Comment by ssorallen** [comment]
Is there evidence that this is causing problems? I'm not opposed to the change if there's proof that this fixes an issue.

File: core-header-panel.css (hunk 1 above)
Comment (ebidel) [inline core-header-panel.css]: Done

**Comment by ebidel** [comment]
PTAL. The evidence is at https://github.com/Polymer/core-scroll-header-panel/issues/37, the but issue occurs on this element too. Entire page repaints are no good, and we should prevent them.  We discovered this in a demo app someone is building.

File: core-header-panel.css [hunk 2]
```diff
@@ -31,6 +31,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   -webkit-overflow-scrolling: touch;
 }
 
+#mainContainer {
```
Comment (ssorallen) [inline core-header-panel.css]: There's a `#mainContainer` selector right above this one. Can they be collapsed into one?

**Comment by ssorallen** [comment]
K sounds good to me. I had one more comment, but otherwise :+1:.

File: core-header-panel.css (hunk 2 above)
Comment (ebidel) [inline core-header-panel.css]: Doh! Done

**Pull request merged**